src/block/test_block_header.cairo

```diff
@@ -147,23 +147,23 @@ func test_read_block_header_validation_context{range_check_ptr, bitwise_ptr: Bit
     // Sanity check for prev_timestamps of next_state
     // First element is the timestamp of this block
     with_attr error_message("The first element does not correspond to the timestamp of the block.") {
-        assert next_state.prev_timestamps[0] = 1415239972;
+        assert 1415239972 = next_state.prev_timestamps[0];
     }
     // Our 2nd element should be the 1st of the dummy
     with_attr error_message("The state does not correspond with the dummy.") {
```

Comment:
This would look better with one `with_attr` block around all asserts, no?

```suggestion
    // Our 2nd element should be the 1st of the dummy
    with_attr error_message("The state does not correspond with the dummy.") {
        assert next_state.prev_timestamps[1] = 0;
    // ...
        assert next_state.prev_timestamps[7] = 6;
    // ...
        assert next_state.prev_timestamps[9] = 8;
    // Our 11th element should be the 10th of the dummy
        assert next_state.prev_timestamps[10] = 9;
    }

```